packages/pangea-sdk/.sdk-ci.yml

```diff
@@ -6,6 +6,15 @@
   rules:
     - if: $CI_PIPELINE_SOURCE == "push"
 
+.pangea-sdk-gcp:
+  before_script:
+    - source ./dev/replace_aws_with_gcp_env_var.sh
```

Comment:
do we need to source this or eval this?

Reply:
We need to source it to execute it in current shell because we need environment variable's updates to persist